source/extensions/quic_listeners/quiche/platform/quic_cert_utils_impl.h

```diff
@@ -16,7 +16,7 @@ class QuicCertUtilsImpl {
   static bool ExtractSubjectNameFromDERCert(QuicStringPiece cert, QuicStringPiece* subject_out);
 
 private:
-  static bool SeekToSubject(QuicStringPiece cert, CBS* tbs_certificate);
+  static bool seekToSubject(QuicStringPiece cert, CBS* tbs_certificate);
```

Comment:
I think this might be an unintended change.

Reply:
ah yep, this is for a different clang-tidy rule soon-to-come. Will revert